### ISSUE TITLE
Image inside the headset shakes/jitters heavily during head movement with Nvidia GPUs

### ISSUE THREAD

**Issue by JellyOsaurus** [alvr-org/ALVR#675]
This is a known issue as pointed out in the discord server, so i thought i would create one here on GitHub. On nvidia hardware, the image inside the headset jitters or shakes heavily on the Linux Build of ALVR server. This will not happen on windows. Someone pointed out this has to do with Nvidia GPUs on Linux not having support for async reprojection in the drivers. 

Maybe somebody smarter than me, can come up with their own workaround to get the posing correct to make the experience more comfortable for the user wearing the HMD on Nvidia.

**Comment by xytovl** [comment]
To be accurate, in order to match video frames with a position, the linux build relies on a hack and unwinds the stack in vkQueuePresent in order to find a TrackedDevicePose_t object and associates it to the video frame.

Unfortunately, this trick only works with async reprojection, which is not supported on nvidia drivers. In the non-async codepath, the TrackedDevicePose_t objects do not seem to match the image.

**Comment by rullinoiz** [comment]
happened to me on windows 10 with an rx 580, fixed it by tinkering with the settings a little to find a sweet spot

**Comment by KaffeinatedKat** [comment]
@rullinoiz Unfortunately this is a Linux specific issue that cannot be solved by messing with a few settings, because of how the Linux driver is written we have to change actual code to fix this

**Comment by gururise** [comment]
Same exact thing occurring on an AMD Radeon 6800XT

**Comment by kenan1099** [comment]
Apparently this will be fixed in driver 470 https://www.gamingonlinux.com/2021/06/nvidia-driver-470-for-linux-to-include-support-for-async-reprojection

**Comment by Espionage724** [comment]
> Same exact thing occurring on an AMD Radeon 6800XT

I have the same issue I believe with a RX 580.

> happened to me on windows 10 with an rx 580, fixed it by tinkering with the settings a little to find a sweet spot

What settings?

**Comment by XyleticWasTaken** [comment]
Installing the 470 nvidia beta drivers for linux fixed this for me

**Comment by KaffeinatedKat** [comment]
> Installing the 470 nvidia beta drivers for linux fixed this for me

What distro are you using? I still get the jitter even with the 470 driver installed

**Comment by XyleticWasTaken** [comment]
Manjaro linux, linux 5.12 kernel (linux-tkg-pds)
alvr settings are 75% resolution, 72hz, reduce color banding on, throttled udp

**Comment by MathiewMay** [comment]
> Manjaro linux, linux 5.12 kernel (linux-tkg-pds)
> alvr settings are 75% resolution, 72hz, reduce color banding on, throttled udp

I can confirm that this is still an issue.
I am on Manjaro with kernel 5.12.19-1 and nvidia driver version 470.57.02 with the same settings as you on v16-rc2, the jitter is still there.

**Comment by nath67** [comment]
> Installing the 470 nvidia beta drivers for linux fixed this for me

Using Arch and on Nvidia driver 470 but it's not better for me.

**Comment by nath67** [comment]
Disabling async reprojection reduce shaking a lot.
It's located at _~/.steam/steam/config/steamvr.vrsettings_ :
```
{
   "steamvr" : {
      "enableLinuxVulkanAsync" : false
   }
}
```

**Comment by Leeo97one** [comment]
Still seems to be an issue with driver version 510.47.03 and NVENC.

**Comment by Leeo97one** [comment]
Is the last release (https://github.com/alvr-org/ALVR/releases/tag/v18.0.0) supposed to fix it?

**Comment by Leeo97one** [comment]
Sounds great, I'll test this ASAP.

**Comment by zarik5** [comment]
Fixed on ALVR v18.0.0
